src/mixins/createCachedRenderMixin.ts

```diff
@@ -121,9 +126,15 @@ const createCachedRenderMixin: CachedRenderFactory = createStateful
 					props[key] = cachedRender.listeners[key];
 				}
 				const classes: { [index: string]: boolean; } = {};
+				const widgetClasses: string[] = widgetClassesMap.get(cachedRender) || [];
```

Comment:
No, just in the initializer do something like this:

```typescript
(instance, options) => {
    widgetClassesMap.set(instance, []);
    instance.own({
        destroy() {
            widgetClassesMap.delete(instance);
        }
    });
}
```

And then you can always just do the following here:

```typescript
const widgetClasses = widgetClassesMap.get(cachedRender);
```

And then you don't ever need to do a `widgetClassesMap.set` below, because you are dealing with a primitive.